package.json

```diff
@@ -65,81 +94,131 @@
+    "prop-types": "^15.5.8",
+    "puppeteer": "^5.3.1",
     "raf": "^3.4.1",
     "react": "^16.8.5",
     "react-dom": "^16.8.5",
     "react-test-renderer": "^16.4.2",
     "regenerator-runtime": "^0.13.2",
+    "rimraf": "^3.0.2",
+    "semantic-release": "^17.1.1",
+    "sinon": "^9.2.0",
+    "style-loader": "^2.0.0",
     "stylelint": "^11.0.0",
     "stylelint-config-terra": "^3.2.0",
     "terra-aggregate-translations": "^1.0.0",
     "terra-dev-site": "^6.5.0",
     "terra-disclosure-manager": "^4.9.0",
     "terra-enzyme-intl": "^3.0.0",
+    "terra-theme-context": "^1.0.0",
     "terra-toolkit": "^6.2.0",
+    "url-loader": "^4.1.1",
     "webpack": "^4.30.0",
-    "webpack-cli": "^3.3.2",
-    "webpack-dev-server": "^3.3.1"
+    "webpack-cli": "^3.3.12",
+    "webpack-dev-server": "^3.3.1",
+    "webpack-merge": "^4.2.2"
+  },
+  "scripts": {
+    "build": "webpack --config ./packages/carbon-graphs/webpack/webpack.config.js --env.TYPE=LIB",
+    "build:dist": "webpack --config ./packages/carbon-graphs/webpack/webpack.config.js --env.TYPE=DIST",
+    "bootstrap:hoist": "lerna bootstrap --hoist",
+    "clean": "lerna clean --yes && rm -rf node_modules || true",
+    "clean:install": "npm run clean && npm install",
+    "clean:obsolete-snapshots": "npm test -- -u",
+    "compile": "npm run compile:clean && npm run compile:build",
+    "compile:build": "lerna ls | sed 's/@cerner\\///g' | cut -d ' ' -f1 | xargs -I {} babel packages/{}/src --out-dir packages/{}/lib --copy-files",
+    "compile:clean": "rm -rf packages/**/lib",
+    "compile:src": "lerna run compile",
+    "compile:prod": "rm -rf ./build; NODE_ENV=production webpack -p",
+    "compile:lowlight": "rm -rf ./build; THEME=clinical-lowlight-theme webpack -p",
+    "compile:fusion": "rm -rf ./build; THEME=orion-fusion-theme webpack -p",
+    "danger": "danger ci",
+    "dependency-markdown": "node scripts/dependency-markdown-generator/DependencyMarkdownGenerator.js",
+    "dev": "node packages/carbon-graphs/webpack/dev-server.js",
+    "docs": "node scripts/root-readme-updater/index.js",
+    "docs:peerDependencies": "node scripts/peer-dependency-generator/index.js",
+    "heroku-prebuild": "npx lerna init",
+    "heroku-postbuild": "npm install --only=dev && npm run compile:prod",
+    "jest": "jest",
+    "jest:coverage": "jest --coverage",
+    "link-parent-bin": "link-parent-bin",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "eslint --ext .js,.jsx . --fix",
+    "lint:scss": "stylelint 'packages/**/src/**/*.scss' --syntax scss",
+    "pretest": "npm run lint",
+    "postinstall": "link-parent-bin && npm run compile && npm run bootstrap:hoist",
+    "start": "tt-serve",
+    "start-heroku": "tt-serve-static --port $PORT --site './build' ",
+    "start-prod": "tt-serve --env.disableHotReloading -p",
+    "start-static": "npm run compile:prod && tt-serve-static --site ./build",
+    "test": "npm run lint && npm run jest && npm run wdio",
```

Comment:
i'd add karma to this test command.

Reply:
Fixed. Please got through this [link](https://github.com/cerner/terra-graphs/pull/5/commits/cf193d3d2617419f5f6857e6b1c7b853c964295c#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R216) to see the changes.